README.md

```diff
@@ -34,146 +34,123 @@ Please help us by [Contributing](CONTRIBUTING.md) to the project.
+Then wait for the cluster to come up and verify that kubernetes is done initializing (be patient, it takes a bit):
 
-   ```bash
-   echo "Hello Rook!" > /tmp/rookFS/hello
-   cat /tmp/rookFS/hello
-   ```
+```
+$ kubectl cluster-info
+```
 
-5. Unmount the shared file system (this does **not** delete the data from the cluster)
+If you see a url response, you are ready to go.
 
-   ```bash
-   ./rook filesystem unmount --path /tmp/rookFS
-   ```
+#### Deploy Rook
 
-6. Cleanup the shared file system from the cluster (this **does** delete the data from the cluster)
+Rook can be setup and deployed in Kubernetes by simply deploying the [rook-operator](https://github.com/rook/rook/blob/master/demo/kubernetes/rook-operator.yaml) deployment manifest.
 
-   ```
-   ./rook filesystem delete --name testFS
-   ```
+```
+$ kubectl create namespace rook
+$ kubectl create -f rook-operator.yaml
+```
 
-#### Object Storage
-1. Create an object storage instance in the cluster
+Use `kubectl` to list pods in the rook namespace. You should be able to see the following: 
 
-   ```bash
-   ./rook object create
-   ```
+```
+$ kubectl -n rook get pod
+NAME                            READY     STATUS    RESTARTS   AGE
+mon0                            1/1       Running   0          1m
+mon1                            1/1       Running   0          1m
+mon2                            1/1       Running   0          1m
+osd-n1sm3                       1/1       Running   0          1m
+osd-pb0sh                       1/1       Running   0          1m
+osd-rth3q                       1/1       Running   0          1m
+rgw-1785797224-9xb4r            1/1       Running   0          1m
+rgw-1785797224-vbg8d            1/1       Running   0          1m
+rook-api-4184191414-l0wmw       1/1       Running   0          1m
+rook-operator-349747813-c3dmm   1/1       Running   0          1m
+```
 
-2. Create an object storage user
+#### Create Storage class
+Before Rook can start provisioning storage, a StorageClass needs to be created. This is used to specify the storage privisioner, parameters, admin secret and other information needed for Kubernetes to interoperate with Rook for provisioning persistent volumes.
+Rook already creates a default admin and demo user, whose secrets are already specified in the sample rook-storageclass.yaml. 
 
-   ```bash
-   ./rook object user create rook-user "A rook rgw User"
-   ```
+However, before we proceed, we need to specify the Ceph monitor IPs. You can find them by running this line (you will need `jq`).
 
-3. Get the connection information for accessing object storage
+```
+$ kubectl -n rook get pod mon0 mon1 mon2 -o json|jq .items[].status.podIP
+"10.2.1.68"
+"10.2.2.66"
+"10.2.0.41"
+``` 
 
-   ```bash
-   eval $(./rook object connection rook-user --format env-var)
-   ```
+Add these IPs into the `monitors` param of the `rook-storageclass.yaml`. Also append the port `6790` to each IPs. They need to be comma-separated.
```

Comment:
I think the one liner command using `sed` is pretty long and ugly for this doc. I think the user can easily look at the list of IPs returned and do replace these values on their own.

@DanKerns That is slated for v0.4. We need to use kubernetes services for each monitor pods.